backend/src/auth/auth.module.ts

```diff
@@ -2,10 +2,9 @@ import { forwardRef, Module } from '@nestjs/common';
 import { AuthController } from './auth.controller';
 import { AuthService } from './auth.service';
 import { UserModule } from '../users/user/user.module';
-import { SharedModule } from '../shared/shared.module';
 
 @Module({
-	imports: [forwardRef(() => UserModule), SharedModule],
+	imports: [forwardRef(() => UserModule)],
```

Comment:
Did you change something else?
Because with me it does not compile if i remove the `SharedModule`..

Reply:
I'm assuming you didn't compile this branch but another one?🧐